src/core_plugins/getting_started/public/lib/add_setup_work.js

```diff
@@ -68,14 +67,14 @@ uiRoutes
         });
         notify.error('Please create a new index pattern');
         kbnUrl.change(CREATE_INDEX_PATTERN_ROUTE);
-        return;
+        throw WAIT_FOR_URL_CHANGE_TOKEN;
```

Comment:
@spalger Do you think this throw should just become part of the `kbnUrl.change` function? It seems to me the two would always want to travel together, no?